File: Api/DataModel/GetAbsenceListRequest.cs
namespace DataModel
{
    public class GetAbsenceListRequest
    {
        public int PageNumber { get; set; }
        public int RowsOfPage { get; set; }
        public bool OrderByDesc { get; set; }
        public string OrderByColumnName { get; set; }
        public DateTime StartDate { get; set; }
        public DateTime EndDate { get; set; }
    }
}

File: frontend/src/@api/dto/get-absence-list-request.ts
export class GetAbsenceListRequest {
  pageNumber: number;
  rowsOfPage: number;
  orderByDesc: boolean;
  orderByColumnName: string;
  startDate: Date;
  endDate: Date;

  constructor(pageNumber: number,
    rowsOfPage: number,
    orderByDesc: boolean,
    orderByColumnName: string,
    startDate: Date,
    endDate: Date) {
    this.pageNumber = pageNumber;
    this.rowsOfPage = rowsOfPage;
    this.orderByDesc = orderByDesc;
    this.orderByColumnName = orderByColumnName;
    this.startDate = startDate;
    this.endDate = endDate;
  }
}